util/cpuprofile/cpuprofile.go

```diff
@@ -157,7 +157,9 @@ func (p *parallelCPUProfiler) profilingLoop() {
 	checkTicker := time.NewTicker(DefProfileDuration)
 	defer func() {
 		checkTicker.Stop()
+		logutil.BgLogger().Info("before StopCPUProfile")
```

Comment:
Would "start StopCPUProfile" and "finish StopCPUProfile" be better？ Feel like these "before" and "after" are rarely used?